backend/app/testbot/main.py

```
@@ -0,0 +1,47 @@
+import os
+from kb import get_or_create_kb
+from server import run_server
+
+def knowledge_base_exists(chatbot_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `knowledge_base_exists`

(ANN201)